packages/core/src/constants/relayer.ts

```diff
@@ -28,7 +26,7 @@ export const RELAYER_PROVIDER_EVENTS = {
   error: "error",
 };
 
-export const RELAYER_RECONNECT_TIMEOUT = ONE_SECOND;
+export const RELAYER_RECONNECT_TIMEOUT = 0.1;
```

Comment:
Can we please remove it entirely for the first reconnect or two? After the first one, it can back off and retry every second or 5s like Kotlin does